### PR TITLE
Addition of Readme and edits browser tab

### PR DESCRIPTION
**Pull request by amandagibson**
#### Description
Changes proposed in this pull request:
addition of detailed readme

* adds information in readme
* changes browser tab to be more descriptive
* fixes some issues when compiling
#### Screenshots:
Screenshot 2019-04-21 at 19 23 37

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,74 @@
...
+## Deployment
+We continuously deploy with netlify at https://khronostimetracking.netlify.com/
+
+---
+## Built with
+- [React](https://github.com/facebook/create-react-app)
+- [Axios](https://github.com/axios/axios)
+- [Semantic UI](https://github.com/Semantic-Org/Semantic-UI-React)
+- [Chart js](https://github.com/chartjs/Chart.js)
+- [Moment Timezone]()
```
Comment (LauraRe) [inline README.md]: website?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,74 @@
...
+- [React Datepicker]()
```
Comment (LauraRe) [inline README.md]: same here

File: README.md (hunk 1 above)
Comment (amandagibson) [reply README.md]: can you contribute? :)

File: README.md (hunk 2 above)
Comment (amandagibson) [reply README.md]: plz

File: README.md [hunk 3]
```diff
@@ -1,2 +1,70 @@
...
+ `git clone https://github.com/CraftAcademy/khronos`
+
+## Installing
+Ensure that the dependency versions on your local machine are as specified in the package.json.
+
+Run `yarn install` to install dependencies.
+
+Run `yarn start` to open in localhost.
+
+As a trial you can log in with the following credentials:
```
Comment (LauraRe) [inline README.md]: I am not sure this is the best place to put the trial credentials

**Pull request merged**